tailwind.config.js

```diff
@@ -5,7 +5,12 @@ module.exports = {
     "./src/**/*.{vue,js,ts,jsx,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
+      backgroundImage: {
```

Comment:
This is more of a one-off background image https://tailwindcss.com/docs/background-image#arbitrary-values